### DISCUSSION TITLE
Profile All queires Export info and keep turned on

### DISCUSSION THREAD

**Discussion by TinyTimothy2** [DaxStudio/DaxStudio#1320] [Q&A]
How can i use Dax Studio to profile activity on SSAS server.
I connect to SSAS server.  I am forced to choose a db on SSAS.
I click ALL QUERIES.
And that seems to start capturing data.
2 issues:
I cannot EXPORT the data in the grid.  If I click export, it does .json.
I cannot ALT-A select and copy to excel

The trace does not stay on.  I don't know how to set it for 1 hour 10 mins or all day.
It is impossible for me to tell what keeps it open and has forced it to stop.

**Comment by dgosbell** [comment]
> I cannot EXPORT the data in the grid. If I click export, it does .json.

All the data in the grid and some extra columns we don't display will be in the json file. If you want to bring the data into something like Excel the *Get Data* feature has built-in json support
![image](https://github.com/user-attachments/assets/5134f9e1-e7a3-453b-a095-83cb6b666975)

> I cannot ALT-A select and copy to excel

I should be able to fix this, unfortunately the grids don't have built-in binding support for copy/paste, but I should be able to configure that. In the meantime, export to .json and then using *Get Data* in Excel should work.

> The trace does not stay on. I don't know how to set it for 1 hour 10 mins or all day.

It should stay on until you stop it, but there is no way to set it to run for a certain amount of time.

> It is impossible for me to tell what keeps it open and has forced it to stop.

Have you checked the *Log* tab? If there was an error of some sort that caused the trace to stop it should be captured there.
